ehsm-service/test_ehsm_service.py

```diff
@@ -0,0 +1,85 @@
+import requests 
+import json
+base_url = "http://10.239.127.69:3000/ehsm?Action="

```

Comment:
you can't use the internal hardcode ip address here.
base_url = "http://10.239.127.69:3000/ehsm?Action="

can we use the hostname as instead?  and you need to set the ip as a variable or marco, it's easy to change it later, some thing like this:
base_url = <ehsm_kms-server_ipaddr:port>"/ehsm?Action="

Reply:
It will be fix.